docs/develop/api-reference/sorted-set-collections.md

```diff
@@ -59,7 +87,7 @@ See [response objects](./response-objects.md) for specific information.
 
 </details>
 
-### SortedSetGetScore
+### SortedSetGetScores
```

Comment:
i think we have both singular and plural now?  also the text below reads singular.

Reply:
Fixed in [98e561c](https://github.com/momentohq/public-dev-docs/pull/120/commits/98e561ca4fede287ee9e17c820c44fe4038c644e)